go/vt/topo/zk2topo/zk_conn.go

```diff
@@ -108,7 +109,6 @@ func (c *ZkConn) Get(ctx context.Context, path string) (data []byte, stat *zk.St
 	return
 }
 
-// GetW is part of the Conn interface.
```

Comment:
these comments obviously add no value but I'm pretty sure some linter or the other (maybe gofmt) will complain about this.

Reply:
I deleted the comments because, afaict, they are not part of the `Conn` interface from `go/vt/topo/conn.go` although I may be missing something.

Reply:
:+1:

Reply:
These should have generated lint errors. These comments are required by go coding standards: https://github.com/golang/go/wiki/CodeReviewComments#doc-comments. I know it's silly, but we'll need to add them back in.

Reply:
Added back in